packages/react-instantsearch/src/components/SearchBox.js

```diff
@@ -172,6 +176,10 @@ class SearchBox extends Component {
       </svg>
     );
 
+    const loadingIndicatorComponent = this.props.loadingIndicatorComponent ? (
+      this.props.loadingIndicatorComponent
+    ) : <DefaultLoadingIndicator />;
```

Comment:
We could provide a default props for `loadingIndicatorComponent` with the value `DefaultLoadingIndicator`. It will avoid the need to do the condition.